src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -56,6 +55,9 @@ internal WebAssemblyHostBuilder(WebAssemblyJSRuntimeInvoker jsRuntimeInvoker)
             Configuration = new ConfigurationBuilder();
             RootComponents = new RootComponentMappingCollection();
             Services = new ServiceCollection();
+            Logging = new LoggingBuilder(Services);
+
+            Logging.SetMinimumLevel(LogLevel.Warning);
```

Comment:
If I'm following correctly, this is defining a default, but the developer could still override it by calling `builder.Logging.SetMinimumLevel(somethingElse)` afterwards? If so that's ideal!

Reply:
Yes, this is possible. Based on the recommendation below, I updated BasicTestApp to decrease the min level.